ecl/eclagent/eclagent.cpp

```diff
@@ -563,6 +563,8 @@ EclAgent::EclAgent(IConstWorkUnit *wu, const char *_wuid, bool _checkVersion, bo
             w->setXmlParams(_queryXML);
         updateSuppliedXmlParams(w);
     }
+    IPropertyTree *costs = queryCostsConfiguration();
+    workflowMachineCost = costs ? costs->getPropReal("@workflowMachine", 0.0): 0.0;
```

Comment:
@agent @hthor or @roxie is probably more appropriate.  It isn't just performing workflow - it executes some workflow items.
@richardkchapman what name do you think it should use
e.g.
cost:
   agent: 10